src/System.Drawing.Common/src/System/Drawing/Imaging/Metafile.Unix.cs

```diff
@@ -391,8 +216,8 @@ public static MetafileHeader GetMetafileHeader(Stream stream)
 
         public static MetafileHeader GetMetafileHeader(string fileName)
         {
-            if (fileName == null)
-                throw new ArgumentNullException(nameof(fileName));
+            // Called in order to emulate exception behavior from netfx related to invalid file paths.
+            Path.GetFullPath(fileName);
```

Comment:
This would add an extra entry in the call stack for the `ArgumentNullException` which might make it harder to diagnose, do we really want that?

Reply:
I see this is matching what we do in the Windows code, disregard.